modules/libraries/src/views/full/PackageLib.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@blueprintjs/core'
+import { toast } from 'botpress/shared'
+import React, { FC, useRef, useState } from 'react'
+
+interface Props {
+  name: string
+  version: string
+  axios: any
+}
+
+const sanitizeName = (text: string) =>
```

Comment:
is there a way we extract this sanitizeName function we use pretty much all over the codebase (core and modules) in bp/common ?